README.md

```diff
@@ -39,7 +39,7 @@ var el = {
 ### Converting Javascript Values into Elements
 
 ```javascript
-var minim = require('minim');
+var minim = require('minim').init();
```

Comment:
1. Yes. For example:

  ```js
  var registry = new ElementRegistry();
  module.exports = registry;
  ```

2. Either we expose `ElementRegistry` as a property of itself or we do that as a special case with the default export. E.g:

  ```js
  ElementRegistry.prototype.ElementRegistry = ElementRegistry;
  ```
  or
  ```js
  registry.ElementRegistry = ElementRegistry;
  module.exports = registry;
 ```

Either way you'll be able to:

```js
var minim = require('minim');
var data;

// Use default registry
data = minim.toElements(['abc', 123]);

// Use a custom registry
registry = new minim.ElementRegistry();
data = registry.toElements(['abc', 123]);
```

Note, this requires moving some methods we have floating around at the module level today into the element registry. I think this makes sense though.

I also think every registry should just default to loading the elements included with Minim. The namespace loading mechanism should only apply to namespaces that aren't the base on top of which everything else is built.